pkg/options/options.go

```diff
@@ -17,6 +17,8 @@ const (
 	OptionsWaitBeforeDelete = "WAIT_BEFORE_DELETE"
 	// OptionsRedirectDetach Redirect detach to the node where volume is attached
 	OptionsRedirectDetach = "REDIRECT_DETACH"
+	// OptionsDeviceFuseMount name of fuse mount device
+	OptionsDeviceFuseMount = "DEV_FUSE_MOUNT"
```

Comment:
Fuse for shared volumes is a px specific implementation. libopenstorage doesn't know about it, right? So should this be called DeviceVirtualMount (or something similar) instead?